test/compute_density.jl

```diff
@@ -17,21 +17,22 @@ if mpi_nprocs() == 1  # not easy to distribute
             kwargs = (temperature=testcase.temperature, smearing=DFTK.Smearing.FermiDirac())
             n_bands = div(testcase.n_electrons, 2, RoundUp) + 4
         end
+        occupation_threshold = 1e-5
```

Comment:
That's rather large. Is this on purpose?

Reply:
well, it was the previous value harcoded in `compute_occupation` :sweat_smile: and yes it is on purpose, for testing we don't really need a strong threshold, do we ?